server/room.ts

```diff
@@ -224,7 +225,7 @@ export class Room implements RoomState {
 	_owner: User | null = null;
 	grants: Grants = new Grants();
 	userRoles: Map<Role, Set<number>>;
-	_autoSkipSegments = true;
+	_autoSkipSegmentCategories: Array<Category> = Array.from([]);
```

Comment:
This makes the default value for newly created rooms (both temporary and permanent) empty, when it should be all the categories.